Microsoft.Recognizers.Text.DateTime.Tests/English/TestDatePeriodParser.cs

```diff
@@ -128,10 +132,6 @@ public void TestDatePeriodParse()
                 BasicTestFuture("I'll be out 3/2015", 2015, 3, 1, 2015, 4, 1);
             }
 
-            //BasicTestFuture("I'll leave this summer", 2016, 6, 1, 2016, 9, 1);
-            //BasicTestFuture("I'll leave in summer", 2017, 6, 1, 2017, 9, 1);
-            //BasicTestFuture("I'll leave in winter", 2016, 12, 1, 2017, 3, 1);
-            //BasicTestFuture("I'll leave in winter, 2017", 2017, 12, 1, 2018, 3, 1);
```

Comment:
@rubio41 These tests seem useful. Is this a matter of difference between north and south hemispheres? If so, maybe we should pass that as conf parameter like other cases.
And assume a default (mentioned in the schema).
We can track that by opening a GitHub issue.